index.js

```diff
@@ -42,27 +47,34 @@ const parseEvent = (event) => {
 };
 
 // Prepare the Messages and Options
-const prepareLogs = (eventData) => {
+const prepareLogs = (eventData, raw_event_message) => {
     return eventData.logEvents.map((event) => {
+        const eventMetadata = {
```

Comment:
Does it make sense to include the `RAW_EVENT_MESSAGE` flag in the event metadata too?

Reply:
@matt-march, but it's configuration-flag but `eventData` is everything about incoming `event`

Reply:
Source is fixed though, not based on the event, right?

Reply:
@matt-march, yup!